From d76c12cd6de36479530c635dcb9c35311fa50705 Mon Sep 17 00:00:00 2001
From: Huachao Mao <huachaomao@gmail.com>
Date: Fri, 15 Dec 2017 15:26:08 +0800
Subject: [PATCH 66/91] Fix incorrect yaml code block in pipeline definition

Change-Id: I0181cb4bfce5bf3b486938ef64532df77cb0a764
(cherry picked from commit 47033f5ac1df573a8a9325be8006d485ea9dd1bc)
---
 doc/source/admin/telemetry-data-pipelines.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/source/admin/telemetry-data-pipelines.rst b/doc/source/admin/telemetry-data-pipelines.rst
index 6bf35d3..ebbba2c 100644
--- a/doc/source/admin/telemetry-data-pipelines.rst
+++ b/doc/source/admin/telemetry-data-pipelines.rst
@@ -49,7 +49,7 @@ The meter pipeline definition looks like:
      - name: 'source name'
        meters:
          - 'meter filter'
-       sinks
+       sinks:
          - 'sink name'
    sinks:
      - name: 'sink name'
@@ -132,7 +132,7 @@ Similarly, the event pipeline definition looks like:
      - name: 'source name'
        events:
          - 'event filter'
-       sinks
+       sinks:
          - 'sink name'
    sinks:
      - name: 'sink name'
-- 
2.7.4

